kong/runloop/handler.lua

```diff
@@ -118,6 +118,15 @@ local function get_now()
 end
 
 
+local function prewarm_hostname_dns_timer(premature, host)
+  if premature then
+    return
+  end
+
+  kong.dns.toip(host)
```

Comment:
this call will fail if the configured hostname is not an actual hostname, but an upstream. `toip` does not resolve upstreams, it only does the dns part. Resolving `upstream` names is done in `balancer.lua`.

Reply:
btw: could be that the error just gets ignored, in that case all is well, but better test it first so we do not get unexpected error messages in the logs.